pkg/storage/storage.go

```diff
@@ -152,7 +153,7 @@ func (is *ImageStore) ValidateRepo(name string) (bool, error) {
 		is.log.Error().Err(err).Str("dir", dir).Msg("unable to read directory")
 		return false, errors.ErrRepoNotFound
 	}
-
+	// nolint (gomnd)
```

Comment:
what is this nolint? you are not even touching this file right?

Reply:
Yes I am not touching this file but I got linting error during building, so I tried to fix that